ui/apps/platform/src/Containers/ComplianceEnhanced/ClusterCompliance/ScanConfigs/Wizard/ScanConfigWizardForm.tsx

```diff
@@ -73,24 +74,51 @@ function ScanConfigWizardForm({ initialFormValues }: ScanConfigWizardFormProps):
         }
     }
 
+    function wizardStepChanged() {
+        handleProfilesUpdate();
+        setCreateScanConfigError('');
+    }
+
+    function scrollToAlert() {
+        if (alertRef.current) {
+            alertRef.current.scrollIntoView({
+                behavior: 'smooth',
+                block: 'start',
+            });
+        }
+    }
+
     function onClose(): void {
         history.push(complianceEnhancedScanConfigsPath);
     }
 
     function setAllFieldsTouched(formikGroupKey: string): void {
-        const fields = Object.keys(formik.values[formikGroupKey]);
-        const touchedState = fields.reduce((acc, field) => ({ ...acc, [field]: true }), {});
-        formik.setTouched({ [formikGroupKey]: touchedState });
+        const groupHasNestedFields =
```

Comment:
This is why I prefer the idea of giving each step of the wizard its own individual "header" with subsections for actual form values. That way you could "touch" all fields within a grouping at once and you could avoid this odd check required. Also, if we were to add a new form element to the profiles or clusters step, there'd be no need to restructure formik. Instead you could add the new form field to whatever step it lives in. 

```
// current
export const defaultScanConfigFormValues: ScanConfigFormValues = {
    parameters: {
        name: '',
        description: '',
        intervalType: 'DAILY',
        time: '',
        daysOfWeek: [],
        daysOfMonth: [],
    },
    clusters: [],
    profiles: [],
};

// suggested
export const defaultScanConfigFormValues: ScanConfigFormValues = {
    parameters: {
        name: '',
        description: '',
        intervalType: 'DAILY',
        time: '',
        daysOfWeek: [],
        daysOfMonth: [],
    },
    clusters: {
        selectedClusters: [],
        newFormField: '', -- potential new field
    },
    profiles: {
        selectedProfiles: []
    },
};
```